### PR TITLE
[Issue #66] Adds global retention limit

### PR DESCRIPTION
**Pull request by amoose**
Adds global retention limit of 720 hours
Allows retention limit to be configured per instance <= 720
Adds minimum value validation as well >= 1

### CODE REVIEW

**Comment by jgrevich** [comment]
Looks good. It's been QAd and ready to go!

**Comment by afeld** [comment]
Does this close https://github.com/fugacious/fugacious/issues/66?

**Comment by afeld** [comment]
Also, would it be possible to configure the max for the 18F instance to be 120 hours, per https://github.com/fugacious/fugacious/issues/66#issuecomment-249578786?

File: config/secrets.yml [hunk 1]
```diff
@@ -13,13 +13,16 @@
...
   db_key_base: 92ffd31f3cb355b7252afc4633d60d10e6e2266a5e0b8e456ddf10d3519e4cda2303dbe136d1ae42c168e4ce17ae327c21b0f473cfed22928e4f46700db4d828
+  max_retention_hours: 720
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
   db_key_base: 92ffd31f3cb355b7252afc4633d60d10e6e2266a5e0b8e456ddf10d3519e4cda2303dbe136d1ae42c168e4ce17ae327c21b0f473cfed22928e4f46700db4d828
+  max_retention_hours: <%= ENV['MAX_RETENTION_HOURS'] %>
```
Comment (afeld) [inline config/secrets.yml]: Might want to add a note in the README about this.

**Comment by amoose** [comment]
@afeld yes, 18F instances will be updated with this change ASAP. 
Also, yes, good point about README, will follow up 👍

**Comment by amoose** [comment]
I'm leaving #66 open until this is pushed to fugacious.18f.gov

**Comment by NoahKunin** [comment]
Would it be possible to get a push by the end of next week @amoose / anything I can do to help that?

**Comment by jgrevich** [comment]
If we don't get someone else to do it by then, I will. We had someone in place for that task but they got overloaded with other priorities.

**Pull request merged**